packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -17,8 +17,10 @@ import {
 import { OrgSelect } from '../mongo-select/index';
 import UserMenu from '../user-menu/index';
 
+export const orgNavHeight = 60;
```

Comment:
Out of curiosity, why is this needed?

Reply:
We export the heights of each Nav so that we can account for the offset of the correct Nav(s) when in tandem with the banners and other components that push content down in Cloud.

It also means the component can change height without the risk of breaking the integration